streaming/rdm_atomic.c

```diff
@@ -243,11 +171,16 @@ static int run_op(void)
 	case FI_BXOR:
 	case FI_ATOMIC_WRITE:
 		for (datatype = 0; datatype < FI_DATATYPE_LAST; datatype++) {
-			ret = check_base_atomic_op(op_type, datatype);
-			if (ret == -FI_ENOSYS || ret == -FI_EOPNOTSUPP)
+			ret = check_base_atomic_op(ep, op_type, datatype, count);
+       			if (ret == -FI_ENOSYS || ret == -FI_EOPNOTSUPP) {
```

Comment:
Weird, it doesn't look like that on my editor but there do seem to be a lot of extra spaces in the front for some reason that are probably messing up the indentation.